paper-input.html

```diff
@@ -95,7 +95,12 @@
 
       <content select="[prefix]"></content>
 
-      <label hidden$="[[!label]]">[[label]]</label>
+      <label hidden$="[[!label]]">
+        [[label]]
+        <template is="dom-if" if="[[required]]">
```

Comment:
Use `hidden$="[[!label]]"` instead; it's slightly faster than a `dom-if` in this case